src/Controller/Component/SearchComponent.php

```diff
@@ -147,6 +147,15 @@ protected function _filterParams(): array
                 continue;
             }
 
+            if (is_callable($value)) {
```

Comment:
Lets limit this to a `\Closure` instance.

Reply:
Is that better?
I just thought since we already use
- `Callback` to produce results using your own custom callable function

it seems to be consistent for now.

Reply:
Using `Closure` whenever possible is better since callable is a very wide type in PHP. For empty values checking it's highly unlikely anyone will need something more than a short anonymous function.

Reply:
Also the availability of [first class callable](https://www.php.net/manual/en/functions.first_class_callable_syntax.php) makes it trivial to turn any non callable into a Closure instance.

Reply:
Done :)